react/features/authentication/components/LoginDialog.native.js

```diff
@@ -22,6 +24,11 @@ import styles from './styles';
  */
 type Props = {
 
+    /**
+     * The color-schemed stylesheet of the base/dialog feature.
+     */
+    _colorSchemedDialogStyles: StyleType,
```

Comment:
can we just call this `_dialogStyles`? the fact that they are color-schemed doesn't really impact how they are used here

Reply:
I used this name because we import styles from the dialog feature as well (```inputDialogStyle```) and I wanted a clear distinguishing between the two thing.